src/lib/services/server/contentful/schema.graphql

```diff
@@ -2968,7 +3028,6 @@ type PageMetadata implements Entry & _Node {
   metaDescription(locale: String): String
   metaTitle(locale: String): String
   parent(locale: String, preview: Boolean, where: PageMetadataFilter): PageMetadata
-  recentNewsCollection(limit: Int = 100, locale: String, preview: Boolean, skip: Int = 0): PageMetadataRecentNewsCollection
```

Comment:
This field was the one we accidentally added to `Page details`, right? In other words, I assume that this deletion is intentional since we added this field to the Top Tier and Core Content pages instead?

Reply:
Cool, thanks for the gut-check!